invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v1.0.0.json

```diff
@@ -12,20 +12,47 @@
         },
         "access" : {
           "properties" : {
-            "access_right" : {
+            "record" : {
               "type" : "keyword"
             },
-            "created_by" : {
-              "type" : "long"
+            "files" : {
+              "type" : "keyword"
+            },
+            "owned_by" : {
+              "properties": {
+                "user": {
+                  "type": "keyword"
+                }
+              }
             },
-            "files_restricted" : {
-              "type" : "boolean"
+            "embargo": {
+              "properties": {
+                "active": {
+                  "type": "boolean"
+                },
+                "until": {
+                  "type": "date"
+                },
+                "reason": {
+                  "type": "text"
+                }
+              }
             },
-            "metadata_restricted" : {
-              "type" : "boolean"
+            "grants": {
+              "properties": {
+                "subject": {
+                  "type": "keyword"
+                },
+                "id": {
+                  "type": "keyword"
+                },
+                "level": {
+                  "type": "keyword"
+                }
+              }
             },
-            "owners" : {
-              "type" : "long"
+            "tokens": {
```

Comment:
agree, renaming makes the connection immediately clear
i renamed the field to `grant_tokens` (the underscore for consistency with `owned_by`, and singular `grant` because the plural is already in `tokens` (?))

Reply:
Makes sense :)